core/src/test/java/org/frankframework/jdbc/JdbcTableListenerTest.java

```diff
@@ -554,28 +556,38 @@ public void testParallelGet() throws Exception {
 		if (!env.getDbmsSupport().hasSkipLockedFunctionality()) {
 			listener.setStatusValueInProcess("4");
 		}
+		listener.setMessageField("tVARCHAR");
 		listener.configure();
 		listener.start();
 
 		try(Connection connection = env.getConnection()) {
-			JdbcTestUtil.executeStatement(env.getDbmsSupport(), connection, "INSERT INTO " + TEST_TABLE + " (TKEY,TINT) VALUES (10,1)", null, new PipeLineSession());
+			JdbcTestUtil.executeStatement(env.getDbmsSupport(), connection, "INSERT INTO " + TEST_TABLE + " (TKEY,TINT,TVARCHAR) VALUES (10,1,'Message 1')", null, new PipeLineSession());
 		}
 
 		try (Connection connection1 = env.getConnection()) {
 			connection1.setAutoCommit(false);
+			boolean shouldStillCommitBeforeClose;
 			RawMessageWrapper<String> rawMessage1 = listener.getRawMessage(connection1,null);
-			assertEquals("10",rawMessage1.getRawMessage());
-			if (listener.changeProcessState(connection1, rawMessage1, ProcessState.INPROCESS, "test")!=null) {
+			assertEquals("Message 1", rawMessage1.getRawMessage());
+			if (listener.changeProcessState(connection1, rawMessage1, ProcessState.INPROCESS, "test") != null) {
 				connection1.commit();
+				shouldStillCommitBeforeClose = false;
+			} else {
+				shouldStillCommitBeforeClose = true;
 			}
 
-
 			try(Connection connection = env.getConnection()) {
-				JdbcTestUtil.executeStatement(env.getDbmsSupport(), connection, "INSERT INTO " + TEST_TABLE + " (TKEY,TINT) VALUES (11,1)", null, new PipeLineSession());
+				JdbcTestUtil.executeStatement(env.getDbmsSupport(), connection, "INSERT INTO " + TEST_TABLE + " (TKEY,TINT,TVARCHAR) VALUES (11,1,'Message 2')", null, new PipeLineSession());
 			}
 			RawMessageWrapper<String> rawMessage2 = listener.getRawMessage(new HashMap<>());
-			assertEquals("11",rawMessage2.getRawMessage());
 
+			// Clean connection status before we assert
+			if (shouldStillCommitBeforeClose) {
+				connection1.rollback(); // Make sure connection has no unfinished connection on it before closing. Required for DB2.
+			}
+			connection1.setAutoCommit(true);
```

Comment:
is de autocommit echt nodig?

Reply:
Ik weet het niet zeker, maar omdat het eerder voor deze connection op `false` wordt gezet en het met jouw Narayana PR leek of database tests elkaar beinvloeden, leek het me voor de zekerheid beter het toch weer terug te zitten.

Voor de test is het nodig om de autocommit op `false` te zetten.